pages/screening_practitioner_appointments/subject_datasets.py

```diff
@@ -0,0 +1,59 @@
+from playwright.sync_api import Page, expect
+from pages.base_page import BasePage
+from enum import Enum
+
+
+class SubjectDatasets(BasePage):
+    def __init__(self, page: Page):
+        super().__init__(page)
+        self.page = page
+
+        self.show_dataset_button = self.page.get_by_role(
+            "link", name="Show Dataset"
+        ).click()
+
+        self.save_dataset_button = self.page.locator(
+            "#UI_DIV_BUTTON_SAVE1"
+        ).get_by_role("button", name="Save Dataset")
+
+        self.select_asa_grade_dropdown = self.page.get_by_label("ASA Grade")
+
+        self.select_fit_for_colonoscopy_dropdown = self.page.get_by_label(
+            "Fit for Colonoscopy (SSP)"
+        )
+
+        self.dataset_complete_radio_button_yes = self.page.get_by_role(
+            "radio", name="Yes"
+        )
+
+        self.dataset_complete_radio_button_no = self.page.get_by_role(
+            "radio", name="No"
+        )
+
+    def click_show_datasets(self) -> None:
+        self.show_dataset_button.click()
```

Comment:
Could you change this to self.click(locator)
```suggestion
        self.click(self.show_dataset_button)
```